frontend/src/components/Navbar/NavbarWorksheetSearch.tsx

```diff
@@ -343,6 +342,8 @@ export function NavbarWorksheetSearch() {
                         `Declined friend request: ${selectedOption.value}`,
                       );
                     }
+                    await resolveFriendRequest(selectedOption.value);
```

Comment:
```suggestion
                    // Resolve after add, so that when adding we can verify that request exists in DB
                    await resolveFriendRequest(selectedOption.value);
```